src/lang/lwc_en.properties

```diff
@@ -619,3 +619,13 @@ lwc.time.seconds=seconds
 lwc.time.early=less than a second
 lwc.time.unknown=Not yet known
 lwc.time.past=%time% ago
+
+#Supports single-byte characters only
```

Comment:
What do you mean by this comment? Is this an issue?

Reply:
Format collapses.
This problem is caused by `Java.lang.String`.

Single byte:
![image](https://user-images.githubusercontent.com/5967134/72337470-f865b280-3705-11ea-8cac-c6ddbaf34c39.png)

Multi byte:
![image](https://user-images.githubusercontent.com/5967134/72337601-32cf4f80-3706-11ea-8df5-3a5edb9cbee2.png)

Reply:
Ah ok, so it's a String.format issue. Got it.